projects/Panoptic-DeepLab/README.md

```diff
@@ -9,11 +9,23 @@ Bowen Cheng, Maxwell D. Collins, Yukun Zhu, Ting Liu, Thomas S. Huang, Hartwig A
 </div><br/>
 
 ## Installation
+
 Install Detectron2 following [the instructions](https://detectron2.readthedocs.io/tutorials/install.html).
 
-## Training
+If using the cityscape:
+- download the dataset [here](https://www.cityscapes-dataset.com)
+- convert images in the set to panoptic by running [this script](https://github.com/mcordts/cityscapesScripts/blob/master/cityscapesscripts/preparation/createPanopticImgs.py)
```

Comment:
it's already in https://detectron2.readthedocs.io/tutorials/builtin_datasets.html#expected-dataset-structure-for-cityscapes.
You can add a link to it instead.

Reply:
done!